utils/oscap-docker.in

```diff
@@ -42,39 +45,40 @@ if __name__ == '__main__':
                                      epilog='See `man oscap` to learn \
                                      more about OSCAP-ARGUMENTS')
     parser.add_argument('--oscap', dest='oscap_binary', default='', help='Set the oscap binary to use')
+    parser.add_argument('--disable-atomic', dest='noatomic', action='store_true', help="Force to use native docker API instead of atomic")
```

Comment:
```suggestion
    parser.add_argument('--disable-atomic', dest='noatomic', action='store_true', help="Force to use native Docker API instead of Atomic")
```